### PR TITLE
Grammar Updates

### PR DESCRIPTION
**Pull request by krishnathanki**
Few Grammar updates were made!

### CODE REVIEW

**Review by KiaWeb — APPROVED** [review]

**Comment by KiaWeb** [comment]
I think these changes are good.

File: README.md [hunk 1]
```diff
@@ -31,8 +31,7 @@ We're in the process of moving all of our work on the STL to GitHub. Current sta
 * Build System: **In progress.** We're working on a CMake build system, which is currently capable of building one
 flavor of the STL (native desktop). We need to extend this to build all of the flavors required for the MSVC toolset
 (e.g. `/clr`, `/clr:pure`, OneCore, Spectre). Until that's done, we're keeping our legacy build system around in the
-`stl/msbuild` subdirectory. (We're keeping those files in this repo, even though they're unusable outside of Microsoft,
-because they need to be updated whenever source files are added/renamed/deleted. We'll delete the legacy machinery as
+`stl/msbuild` subdirectory. (We're keeping those files in this repo, even though they're unusable outside of Microsoft because they need to be updated whenever source files are added/renamed/deleted. We'll delete the legacy machinery as
```
Comment (frederick-vs-ja) [inline README.md:34]: Please break this line.

File: README.md [hunk 2]
```diff
@@ -115,7 +114,7 @@ You can report STL bugs here, where they'll be directly reviewed by maintainers.
 
 **Please help us** efficiently process bug reports by following these rules:
 
-* Only STL bugs should be reported here. If it's a bug in the compiler, CRT, or IDE, please report it through Developer
+* Only STL bugs should be reported here. If it's a bug in the compiler, CRT, or IDE, please report it through the Developer
```
Comment (frederick-vs-ja) [inline README.md:117]: And this line.

File: README.md [hunk 3]
```diff
@@ -247,15 +246,15 @@ CTest will only display the standard error output of tests that failed. In order
 
 ## Running A Subset Of The Tests
 
-`${PROJECT_BINARY_DIR}\tests\utils\stl-lit\stl-lit.py` can be invoked on a subdirectory of a testsuite and will execute
-all the tests under that subdirectory. This can mean executing the entirety of a single testsuite, running all tests
+`${PROJECT_BINARY_DIR}\tests\utils\stl-lit\stl-lit.py` can be invoked on a subdirectory of a test suite and will execute
+all the tests under that subdirectory. This can mean executing the entirety of a single test suite, running all tests
```
Comment (StephanTLavavej) [inline README.md:250]: Adding a space to "test suite" seems reasonable (it doesn't seem to be used as a single word very often, there's no strong argument for doing so unlike "spacetime", and this aligns with [LLVM lit's documentation](https://llvm.org/docs/CommandGuide/lit.html) which almost exclusively uses a space), but there are more occurrences that should be updated.

File: README.md [hunk 4]
```diff
@@ -277,7 +276,7 @@ testsuite fails, CTest will simply report that the `stl` test failed.
 
 Example:
 ```
-0% tests passed, 1 tests failed out of 1
+0% of tests passed, 1 test failed out of 1
```
Comment (StephanTLavavej) [inline README.md:279]: This is verbatim script output that must not be changed.

**Review by StephanTLavavej — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Thanks! I've pushed a conflict-free merge with `main`, reverted several changes (see my comments for each rationale), and changed all remaining occurrences of "testsuite" to "test suite". I believe that these changes are now an improvement.

**Comment by StephanTLavavej** [comment]
I'm *speculatively* mirroring this to the MSVC-internal repo - please notify me if any further changes are pushed.

**Review by CaseyCarter — APPROVED** [review]

**Comment by StephanTLavavej** [comment]
Thanks for these README grammar improvements, and congratulations on your first microsoft/STL commit! :tada: :scroll: :smile_cat:

**Pull request merged**